src/components/crownConfig.vue

```diff
@@ -204,7 +182,25 @@ export default {
 
       this.$emit('save-state');
     },
+    repositionCrown() {
+      const shapeView = this.shape.findView(this.paper);
+
+      if (!shapeView) {
```

Comment:
Boundary events and intermediate events are now placing the crown at the end of the label on initial drop.

Reply:
If you drag an escalation event from a call activity to a task, the crown moves to the label after error.

Reply:
Looks like there's a `useModelGeometry` option that can be passed to `getBBox` that fixed this:
https://resources.jointjs.com/docs/jointjs/v3.1/joint.html#dia.ElementView.prototype.getBBox

That's super useful!